apollo-runtime/src/main/java/com/apollographql/apollo/internal/RealApolloCall.java

```diff
@@ -210,61 +211,69 @@ private RealApolloCall(Builder<T> builder) {
         .build();
   }
 
-  final class AsyncCall implements ApolloInterceptor.CallBack {
-
-    private final Callback<T> responseCallback;
-
-    private AsyncCall(Callback<T> callback) {
-      this.responseCallback = callback;
-    }
-
-    @Override public void onResponse(@Nonnull final ApolloInterceptor.InterceptorResponse response) {
-      try {
+  @Nonnull @Override public Operation operation() {
+    return operation;
+  }
 
-        if (responseCallback == null) {
-          return;
-        }
-        if (canceled) {
-          responseCallback.onCanceledError(new ApolloCanceledException("Canceled"));
-          return;
-        }
-        if (queryFetcher.isPresent()) {
-          queryFetcher.get().refetchAsync(new QueryFetcher.OnFetchCompleteCallback() {
-            @Override public void onFetchComplete() {
-              //noinspection unchecked
-              responseCallback.onResponse(response.parsedResponse.get());
-            }
-          });
-        } else { //noinspection unchecked
-          responseCallback.onResponse(response.parsedResponse.get());
+  private void refetchQueryWatchers() {
```

Comment:
'refetchQueryWatchers` sort of makes it seem like we will be refetching all query watchers - but really we are refetching the set of "refetch" query watchers, right?

It's an internal method, so not a big deal at all.